js/main.js

```diff
@@ -0,0 +1,9 @@
+// main.js
+// Contains Javascript functions.....
+

```

Comment:
Please add `"use strict";`
see below for more information
https://www.w3schools.com/js/js_strict.asp